packages/client.api/src/queries/Queries.ts

```diff
@@ -45,7 +45,7 @@ export type QuerySignatureFromDef<T extends QueryDefinition<any, any>> =
 
 export type QueryParameterType<
   T extends Record<any, QueryDataTypeDefinition<any, any>>,
-> = NOOP<PartialByNotStrict<NotOptionalParams<T>, OptionalQueryParams<T>>>;
```

Comment:
Why remove the NOOP? I remember when you explained this to me, it was to help expand the type out right,e.g. when you hovered over it, or am I misremembering?

Reply:
NOOP, while producing a simpler intellisense was also stripping out the mapped jsdoc. We will have to find another way possibly!